src/main/webapp/WEB-INF/tags/wrapper.tag

```diff
@@ -150,14 +150,24 @@
     <jsp:doBody/>
 </main>
 
+<spring:eval expression="@environment.getProperty('features.external-cookie-link.enabled')" var="useExternalCookieLink" />
+<spring:eval expression="@environment.getProperty('features.external-cookie-link.url')" var="externalCookieLink" />
+
 <footer class="group js-footer" id="footer" role="contentinfo">
     <div class="footer-wrapper">
         <div class="footer-meta">
             <div class="footer-meta-inner">
                 <h2 class="visuallyhidden"><spring:message code="public.template.footer.support.links" /></h2>
                 <c:set var="footerUrl" value="https://hmcts-access.service.gov.uk" />
                 <ul>
-                    <li><a href="${pageContext.request.contextPath}/cookies"><spring:message code="public.template.footer.support.link.cookies" /></a></li>
+                    <c:choose>
+                        <c:when test="${useExternalCookieLink}">
+                            <li><a href="${externalCookieLink}"><spring:message code="public.template.footer.support.link.cookies" /></a></li>
```

Comment:
I think we should actually have a redirect from our `/cookies` to the confgured value (for reference - have a look at how the contactUs redirect is happening). 
The rationale is to keep any external links there might be to hmcts-access.service.gov.uk/cookies pointing to the right content, if we decide to stop updating that page and host it externally.